lib/stbcbckt.gi

```diff
@@ -2955,24 +2955,41 @@ end );
 InstallMethod( Intersection2, "perm groups", IsIdenticalObj,
   [ IsPermGroup, IsPermGroup ], 0,
 function( G, H )
-local   Omega,  P,  rbase,  L,mg,mh,i;
-    
+local   Omega, P, rbase, L, mg, mh, IsFinished, mg_minus_mh, mh_minus_mg;
```

Comment:
This is minor, but I'd write is_finished instead of IsFinished, just because readers of GAP code will probably think `IsFinished` is an attribute.

Reply:
I agree that in general local variables definitely should not start with upper case letters, I only make exceptions for single-letter variables.

Then again, this function uses `Omega` as a local variable name (which I think is a bad idea, esp. as there is a global function `Omega`). So... 🤷 

(BTW, I'd use `isFinished` or `done` or `isdone`... but this is splitting hairs, and I am pretty sure that @ChrisJefferson would agree with me when I see we won't block a PR over such a minuscule "issue" :-) .